hvac/v1/__init__.py

```diff
@@ -1223,6 +1227,10 @@ def transit_verify_signed_data(self, name, input_data, algorithm=None, signature
             params['context'] = context
         if prehashed is not None:
             params['prehashed'] = prehashed
+        if sign_algorithm is not None:
+            params['signature_algorithm'] = sign_algorithm
+        else:
+            params['signature_algorithm'] = "pss"
```

Comment:
If this is going to be default, please just set it as such in the method signature. I.e., `, sign_algorithm = None):` => `, sign_algorithm='pss'):`

Reply:
will do